ethcore/client-traits/src/lib.rs

```diff
@@ -173,6 +177,10 @@ pub trait EngineClient: Sync + Send + ChainInfo {
 pub trait ImportBlock {
 	/// Import a block into the blockchain.
 	fn import_block(&self, block: Unverified) -> EthcoreResult<H256>;
+
+	// todo[dvdplm]: probably a bad place for this method
```

Comment:
seems reasonable to me at least or maybe introduce another trait `ImportVerifiedBlock`

Reply:
The reason I was hesitant is the comment above saying `// FIXME Why these methods belong to BlockChainClient and not MiningBlockChainClient?` – I don't really know who/when left it and I think it's not relevant any more (even before my changes) but I'm also not 100% sure.

Reply:
Right, the FIXME is a bit vague.  I think it refers to [this](https://github.com/paritytech/parity-ethereum/blob/0dfe3c9ffacfd4b3f1d6bf46031cb26ff2244b2e/ethcore/client-traits/src/lib.rs#L212-#L213).

`BlockChainClient` depends on `ImportBlock` thus the FIXME has nothing to do with `ImportBlock` itself but that it is a super trait to `BlockChainClient`. I suggest we remove the FIXME.